XenAdmin/ConsoleView/VNCTabView.cs

```diff
@@ -501,7 +508,17 @@ private void guestMetrics_PropertyChanged(object sender, PropertyChangedEventArg
         {
             if (e.PropertyName == "other")
             {
+                if (RDPEnabled)
+                {
+                    if (vncScreen.UseVNC && (tryToConnectRDP || (!vncScreen.UserWantsToSwitchProtocol && Properties.Settings.Default.AutoSwitchToRDP)))

```

Comment:
Could you explain in words what all the variables on this line mean? I'm wondering if it should just read 
`if (vncScreen.UseVNC && (tryToConnectRDP || Properties.Settings.Default.AutoSwitchToRDP))`
but I'm not clear what the meaning of all the variables are.